### PR TITLE
Should fix the end of term/time bug :)

### PR DESCRIPTION
**Pull request by mstratford**
Fixes: #38 

I've added a check to make sure that if there is only one show left scheduled (the last show of a term), it doesn't try and get a non-existent show for the 2nd Up Next line :)

I guess we will have to remember and see if it works at the end of the term 👍

### CODE REVIEW

**Comment by mstratford** [comment]
I have tested locally that this addition doesn't cause Timelord to break in normal use.

File: src/scripts/timelord.js [hunk 1]
```diff
@@ -494,9 +494,15 @@ window.Timelord = {
 		if (!shows || shows[0] === null) {
 			Timelord._$('#next-shows').addClass('hidden');
 		} else {
+			var numNextShows = 2;
+			if (shows[1] === null) {
+				numNextShows = 1;
+				Timelord._$('#next-shows #next1').addClass('hidden');
+			}
 			Timelord._$('#next-shows').removeClass('hidden');
+			Timelord._$('#next-shows #next1').removeClass('hidden');
```
Comment (LordAro) [inline src/scripts/timelord.js]: Sure this works as intended? Looks to me like the class gets added above, then gets removed here regardless of the condition..

Comment (mstratford) [inline src/scripts/timelord.js]: Oops, I'll take another look.

Comment (mstratford) [inline src/scripts/timelord.js]: Should be fixed now. I've also commented it a bit.

**Pull request merged**